requirements/dev.txt

```diff
@@ -36,7 +36,7 @@ click-log==0.4.0
     # via edx-lint
 code-annotations==1.3.0
     # via edx-lint
-cryptography==39.0.2
+cryptography==40.0.2
```

Comment:
Contains some [breaking changes](https://cryptography.io/en/latest/changelog/#v40-0-0) but nothing affecting the usage of package in current repo.